### ISSUE TITLE
cli: `instance create` home page

### ISSUE THREAD

**Issue by jirikuncar** [inveniosoftware/invenio-base#37]
- [ ] ask which endpoint should be registered as `/`
- [ ] otherwise create view with `Welcome to {{ site_name }}!`

**Comment by lnielsen** [comment]
I would rather add an independent module ``Invenio-Frontpage`` and instead of adding the view to the cookiecutter template in ``Invenio-Base``.

1) In my opinion it's best to keep the cookiecutter template with as little code as possible.
2) The front-page view will likely need to depend on e.g. search and other related modules with variables that need to be injected into the template. For novice users, that likely requires them to know the modules API.
2) The front-page view could be subject to significant number of changes which would impact release cycle of Invenio-Base.

**Comment by tiborsimko** [comment]
I like the idea of addressing this elsewhere notably to profit from the point no.2 (depending on search).

Do we need another `invenio-frontpage` module though?  Cannot we address this inside `invenio-collections` or `invenio-communities`?  The idea being that "collections" or "communities" also provide access points for their members, and one can consider the Invenio home page to be simply the general access point to the "guest community"...

**Comment by lnielsen** [comment]
Probably we can avoid it an ``invenio-frontpage`` but how do we deal with competing blueprints (say both collections, search, communities provide frontages).

**Comment by tiborsimko** [comment]
One possible option: we require communities (they provide access points, so our splash pages); the collections are only facets inside given communities (so they don't provide any splash pages); and the search is providing only the search results UI (so no home page).

**Comment by tiborsimko** [comment]
(and if we don't want to promote `invenio-comunities` to a status of required module, which I think we don't want to do, then perhaps `invenio-pages` could take care of this; in any case it seems interesting to leave out `invenio-collections` to their role of facets)

**Comment by jirikuncar** [comment]
> ask which endpoint should be registered as `/`

I'm still not convinced about any other solution :wink: 

```console
$ inveniomanange instance create
...
root_endpoint [None]:  # empty endpoint name will generate default Welcome page.
```

**Comment by jirikuncar** [comment]
Closed via #58.
